app/src/main/java/com/x8bit/bitwarden/data/platform/repository/BridgeRepository.kt

```diff
@@ -0,0 +1,27 @@
+package com.x8bit.bitwarden.data.platform.repository
+
+import com.bitwarden.bridge.model.SharedAccountData
+
+/**
+ * Provides an API for querying disk sources required by [BridgeRepositoryImpl].
```

Comment:
I wouldn't refer to an `...Impl` here. Just say this is required for the Bridge Service functionality?

Reply:
121a762c79d99b01de03b1c78f0f4cebd7e88d96